### PR TITLE
[CONTENT] Two versus One Fight Training Patrol

### PR DESCRIPTION
**Pull request by tobes03**
## About The Pull Request

Fun two-on-one fighting patrol for the general training file with plenty of stat-locked outcomes. A few skill locked outcomes, although almost exclusively for fighter cats. I took your idea, scribble, if you're reading this <3, of how cats with different traits and the Fighter skill would still have different fighting styles.

## Why This Is Good For ClanGen

Fun patrol! Particularly to acknowledge that having three cats on a patrol would make things unbalanced in a spar.

## Proof of Testing

image
image
image

### CODE REVIEW

File: resources/dicts/patrols/general/training.json [hunk 1]
```diff
@@ -12069,5 +12069,414 @@
...
+	"min_max_status": {
+		"normal adult": [3, 3]
+	},
+	"weight": 20,
+	"chance_of_success": 30,
+	"intro_text": "Since they have three cats, p_l suggests they practice two-on-one fighting.",
+	"decline_text": "r_c announces {PRONOUN/r_c/subject} {VERB/r_c/want/wants} to practice stalking instead.",
+	"success_outcomes": [
+        {
+            "text": "p_l organizes the patrol into a spar, and the cats congratulate after each other on a good fight.",
```
Comment (scribblecrumb) [inline resources/dicts/patrols/general/training.json]:
```suggestion
            "text": "p_l organizes the patrol into a spar, and the cats congratulate each other on a good fight.",
```

File: resources/dicts/patrols/general/training.json [hunk 2]
```diff
@@ -12069,5 +12069,414 @@
...
+            "relationships": [
+        {
+                    "cats_to": ["patrol"],
+                    "cats_from": ["patrol"],
+                    "mutual": false,
+                    "values": ["comfort", "respect", "platonic"],
+                    "amount": 7
+        }
+        ],
+        "art": "patrol_outcome_art"
```
Comment (scribblecrumb) [inline resources/dicts/patrols/general/training.json]: looks like you forgot to choose art here?

File: resources/dicts/patrols/general/training.json [hunk 3]
```diff
@@ -12069,5 +12069,414 @@
...
+                    "amount": 10
+                }
+        ],
+        "art": "train_general_intro"       
+        },
+        {
+            "text": "p_l suggests that {PRONOUN/p_l/subject} and r_c team up against s_c. s_c warily agrees, but {PRONOUN/s_c/subject} {VERB/s_c/have/has} nothing to worry about. Even outnumbered, s_c proves {PRONOUN/s_c/poss} talent in battle, and r_c and p_l heap praise on {PRONOUN/s_c/object} after the bout.",
+            "exp": 10,
+            "weight": 40,
+            "stat_trait": [ "gloomy", "lonsome", "insecure", "nervous" ],
```
Comment (scribblecrumb) [inline resources/dicts/patrols/general/training.json]:
```suggestion
            "stat_trait": [ "gloomy", "lonesome", "insecure", "nervous" ],
```

**Review by scribblecrumb — CHANGES_REQUESTED** [review]

File: resources/dicts/patrols/general/training.json (hunk 1 above)
Comment (tobes03) [reply resources/dicts/patrols/general/training.json]: sometimes i don't even understand the suggested change at first bc i can't process that i wrote something so incoherent 😭

**Review by scribblecrumb — APPROVED** [review]

**Pull request merged**